dashboard/app/views/levels/_teacher_panel.html.haml

```diff
@@ -31,7 +31,7 @@
         - send("#{"playlab"}_project_view_projects_url".to_sym, channel_id: example, host: 'studio.code.org', port: 443, protocol: :https)
       - else
         - send("#{@level.game.app}_project_view_projects_url".to_sym, channel_id: example, host: 'studio.code.org', port: 443, protocol: :https)
-  - elsif @level.ideal_level_source_id && @script # old style 'solutions' for blockly-type levels
+  - elsif @script && (@level.ideal_level_source_id || @script.csf?)  # old style 'solutions' for blockly-type levels
```

Comment:
Do we need the csf check here since its not checking for authorized teacher?

Reply:
i didn't dig too deep into the meaning of `@level.ideal_level_source_id`, so i just figured that in the interest of making sure we always provide examples for CSF courses that it would be safest to include the check here as well.

if you think it would never be the case that we wouldn't have a `@level.ideal_level_source_id` but that `@script.csf?` would be true, i should probably remove this check for clarity.

Reply:
I think removing the check is probably good. There might be something that is not csf but has the old style solutions so just to be safe lets not change it.

Reply:
I agree with Dani - since this isn't currently deciding to show the solution based on whether the teacher is authorized, it's not explicitly excluding csf teachers, so I would leave it as it was.

Reply:
👍